torch/onnx/symbolic_opset9.py

```diff
@@ -1222,7 +1230,7 @@ def abs(g, self):
 
 
 def log(g, self):
-    return g.op("Log", self)
+    return sym_help._unary_ufunc_helper(g, self, "Log")
 
 
 def log1p(g, self):
```

Comment:
log1p is a unary floating ufunc on your list but the ONNX export isn't modified. Is it working as expected?

Reply:
I didn't modify, assuming it calls `log` already. The same reason we don't need to modify `rsqrt`? https://github.com/pytorch/pytorch/blob/e0373d504746f2d5d0b0187e4b51f3a67b7f9e63/torch/onnx/symbolic_opset9.py#L217

All the tests pass though (and `log1p` is listed in the list in test_pytorch_onnx_onnxruntime.py file).

Should log1p export be modified? I'm not sure why and how.

Reply:
SGTM.